components/GraphPane/GraphPane.scss

```diff
@@ -17,3 +17,11 @@
 #colorize-key {
   margin-top: 1rem;
 }
+
+#report-section {
+  // Hide empty sections
+  h3:has(+ h3),
+  h3:last-child {
+    display: none;
+  }
+}
```

Comment:
I think it's not easy to detect whether a section is empty in React, so I thought I'd just wrap them in a div and use CSS.